app/assets/v2/js/pages/profile.js

```diff
@@ -39,43 +44,52 @@ $(document).ready(function() {
       fetchInProgress = true;
       loadingImg.className = loadingImg.className.replace('hidden', 'visible');
 
-      fetch(location.href.replace(location.hash, '') + '?p=' + (++page) + '&a=' + activityName).then(
```

Comment:
There are still a lot of arbitrary style changes here, please revert them to the way they were